pkg/client/v1/proto/storage_service.proto

```
@@ -77,6 +77,9 @@ message GetProfileResponse {
 
   // NetworkNeighborhood is populated when kind is "networkNeighborhood"
   github.com.kubescape.storage.pkg.apis.softwarecomposition.v1beta1.NetworkNeighborhood network_neighborhood = 5;
+
+  // ContainerProfile is populated when kind is "containerProfile"
+  github.com.kubescape.storage.pkg.apis.softwarecomposition.v1beta1.ContainerProfile container_profile = 6;
```

Buf (1.68.4)

[error] cannot find `github.com.kubescape.storage.pkg.apis.softwarecomposition.v1beta1.ContainerProfile` in this scope

(COMPILE)